### PR TITLE
bug fix: labelSelectorForVolume

### PR DESCRIPTION
**Pull request by 7ing**
labelSelector should add req and assgin back.

fixes: #24

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -18,19 +18,56 @@ require (
 )
 
 require (
+	github.com/beorn7/perks v1.0.1 // indirect
```
Comment (7ing) [inline go.mod:21]: auto generated by `go mod tidy -compat=1.17`

Comment (munnerz) [reply go.mod:21]: I think these changes can be excluded from this PR, especially as our presubmits run with Go 1.18 now: https://github.com/jetstack/testing/blob/3bb9af8d51d14f32f6d3961a66e47e370d4d537e/config/jobs/cert-manager/csi-lib/cert-manager-csi-lib-presubmits.yaml#L16

(plus also exclude changes to go.sum as well)

Unless these came about due to the new tests you've added in this PR too? 👀

Comment (7ing) [reply go.mod:21]: Yes, these changes are due to new tests...required during `go test ./...`

**Pull request merged**